src/Factories/FunFunFactory.php

```diff
@@ -712,13 +712,9 @@ private function newHonorificValidator(): AllowedValuesValidator {
 	private function getHonorifics(): Honorifics {
 		return $this->createSharedObject( Honorifics::class,
 			function () {
-				return new Honorifics( [
-					// In the future, this could be locale-specific or filled from a DB table
-					'' => 'Kein Titel',
-					'Dr.' => 'Dr.',
-					'Prof.' => 'Prof.',
-					'Prof. Dr.' => 'Prof. Dr.'
-				] );
+				$json = ( new SimpleFileFetcher() )->fetchFile( $this->getI18nDirectory() . '/data/honorifics.json' );
```

Comment:
Should this be validated through the deprecated `JsonStringReader`?

Reply:
For consistencies sake, yes. Even if that means using a deprecated class and json_decoding twice on every request 😿